src/inference/src/dev/core_impl.cpp

```diff
@@ -102,7 +59,65 @@ ov::CoreImpl::CoreImpl(bool _newAPI) : m_new_api(_newAPI) {
     }
 }
 
-void ov::CoreImpl::register_plugins_in_registry(const std::string& xmlConfigFile) {
+ov::util::FilePath ov::getPluginPath(const std::string& plugin) {
```

Comment:
Could you use snake_case for new methods?

Reply:
I will do it in next PR